### ISSUE TITLE
ANR when switching back to app

### ISSUE THREAD

**Issue by divergentdave** [divergentdave/Transparensbee#22]
```
java.lang.IllegalStateException: attempt to re-open an already-closed object: SQLiteDatabase: /data/user/0/party.davidsherenowitsa.transparensbee/databases/STH_Storage.db
...
at party.davidsherenowitsa.transparensbee.DBStatistics.getServerSuccessFailure(DBStatistics.java:140)
at party.davidsherenowitsa.transparensbee.StatsArrayAdapter.getView(StatsArrayAdapter.java:92)
```

**Comment by divergentdave** [comment]
I should pull some of the stuff out of the DBStatistics constructor (database connection setup, listener registration), put it in a new method, and call that from MainActivity.onStart(), because onStop() is tearing down things that aren't getting set back up without an onStart() override.

**Comment by divergentdave** [comment]
Fixed in dc03474dc1501531f9e339fd8494f1031df75458
